src/styles/_catalog.scss

```diff
@@ -1,9 +1,23 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));
```

Comment:
dont think that its written in right way, you stretch all elements on page with 1fr

Reply:
yes, you are totally right!